canonical/canonical.com
# canonical.com homepage

## Done

- Added tabbed content script. We already had one that handled tab functionality for navigation tabs, but not for [content tabs](https://vanillaframework.io/docs/patterns/tabs#content). I pulled this script from what we have on u.com with slight modifications 
- Updated homepage copy as per [doc](https://docs.google.com/document/d/1GNa1FaTkARdrVwYgHXf5H-6sqOLjkJENDistDDcsi4o/edit)
- Applied new styling

## QA

- Open the [demo](https://canonical-com-898.demos.haus/) and check against doc
- Check CTAs and tab functionality
- Check medium/mobile screen styles

Fixes https://warthogs.atlassian.net/browse/WD-3734

## Prior notes from @lyubomir-popov :

Almost complete, two notes:
- [ ] We need a fade, as in the search component to prevent this: 

![image](https://github.com/canonical/canonical.com/assets/2741678/bebc82f0-b0fe-468a-b9dc-92ee6cc86eaf)

- [ ] Tabs theming copy-pasted from the vanilla pr, will be removed once the theme is in vanilla.
